corehq/apps/settings/urls.py

```diff
@@ -37,7 +37,8 @@
     url(r'^products/', include(product_settings)),
     url(r'^programs/', include(program_settings)),
     url(r'^locations/', include(location_settings)),
-    url(r'^api/id_mapping/$', project_id_mapping, name="project_id_mapping")
+    url(r'^api/id_mapping/$', project_id_mapping, name="project_id_mapping"),
+    url(r'^events/', include('corehq.apps.events.urls')),
```

Comment:
(Not in this PR, but in a later PR ...) how do you feel about making the URLs "a/<domain>/events/events/" and "a/<domain>/events/attendees/" instead of  "a/<domain>/settings/events/" and "a/<domain>/settings/attendees/" (to follow how the code is laid out)?

Reply:
Yeah, I think that's a great idea. I was not 100% where to put this logic really, so thumb-sucked settings route (it felt like the right place at the time).